drivers/ina2xx/ina2xx.c

```diff
@@ -106,8 +116,12 @@ int ina2xx_init(ina2xx_t *dev, const ina2xx_params_t *params)
         return status;
     }
 
-    /* Set calibration so that current register resolution is 0.01 mA */
-    uint32_t calib = 4096000 / params->rshunt_mohm;
+    /* Set calibration register based on the shunt resistor.
+     * ==> Current will be in mA
+     */
+    uint32_t calib = CURRENT_QUOTIENT / params->rshunt_mohm;
+    /* Change resultion from mA to 0.01 mA by multiplying with 100 */
+    calib *= 100;
```

Comment:
Possible loss of precision compared to the previous version?

Reply:
Yes, indeed :-/ Fixed.